src/Faker/Provider/ru_RU/Address.php

```diff
@@ -69,12 +69,47 @@ class Address extends \Faker\Provider\Address
         'Наро-Фоминск', 'Ногинск', 'Одинцово', 'Озёры', 'Орехово-Зуево',
         'Павловский Посад', 'Подольск', 'Пушкино', 'Раменское', 'Сергиев Посад',
         'Серебряные Пруды', 'Серпухов', 'Солнечногорск', 'Ступино', 'Талдом',
-        'Чехов', 'Шатура', 'Шаховская', 'Щёлково',
+        'Чехов', 'Шатура', 'Шаховская', 'Щёлково', 'Благовещенск', 'Владивосток', 
```

Comment:
Do you have a source for this data? Does it comply with the MIT license of the package?